### PR TITLE
docs: add Mandelbrot’s information option

### LINKED ISSUE #74: Document customizing information panel properties
See frctl/fractal#678.

### PR DESCRIPTION
**Pull request by LeBenLeBen**
closes #74

### CODE REVIEW

File: docs/guide/web/default-theme.md [hunk 1]
```diff
@@ -263,6 +263,44 @@ Customize labels used in the templates, useful for translating the interface for
...
+{
+    information: [
+        {
+            label: 'Version',
+            value: require('./package.json').version,
+        },
+        {
+            label: 'Built on',
+            value: new Date(),
+            type: 'time', // Outputed as a <time /> HTML tag
```
Comment (mihkeleidast) [inline docs/guide/web/default-theme.md:280]:
```suggestion
            type: 'time', // outputs a <time /> HTML tag
```
actually not sure what the correct past tense of output is but outputed feels weird - i think present tense makes a bit more sense.

File: docs/guide/web/default-theme.md [hunk 2]
```diff
@@ -263,6 +263,44 @@ Customize labels used in the templates, useful for translating the interface for
...
+    ],
+}
+```
+
+* **Default:**
+
+    ```js
+    [
+        {
+            label: config.labels.builtOn,
```
Comment (mihkeleidast) [inline docs/guide/web/default-theme.md:294]: can we link this to the labels configuration section? this might be confusing as to what it means.

**Review by mihkeleidast — CHANGES_REQUESTED** [review]

File: docs/guide/web/default-theme.md (hunk 1 above)
Comment (LeBenLeBen) [reply docs/guide/web/default-theme.md]: ✅

File: docs/guide/web/default-theme.md (hunk 2 above)
Comment (LeBenLeBen) [reply docs/guide/web/default-theme.md]: ✅

**Review by mihkeleidast — APPROVED** [review]

**Pull request merged**